projects/angular/src/wizard/STYLES.md

```diff
@@ -8,11 +8,14 @@
 | --clr-wizard-stepnav-bgcolor                   | Wizard navigation background color                                                        |
 | --clr-wizard-main-text-color                   | Wizard text color                                                                         |
 | --clr-wizard-stepnav-text                      | Wizard navigation text color                                                              |
-| --clr-wizard-stepnav-text--active              | Wizard navigation text color of the currently active element                              |
+| --clr-wizard-stepnav-text-active               | Wizard navigation item color when active                                                  |
+| --clr-wizard-stepnav-text-hover                | Wizard navigation item color when hovered                                                 |
+| --clr-wizard-stepnav-text--active              | Wizard navigation item color when selected (misnamed)                                     |
```

Comment:
We can add deprecation for `--active` properties in favor of `selected` and add thme for removal in v18.

```suggestion
| --clr-wizard-stepnav-text--active              | Deprecated in v17 in favor of `--clr-wizard-stepnav-text-selected`                                     |
| --clr-wizard-stepnav-text-selected              | Wizard navigation item color when selected                                   |
```

Reply:
No, I don't want to introduce a new variable and make the code support both. I'd rather just rename it v18.